config/examples/Creality/CR-10 S5/BigTreeTech SKR Mini E3 1.2 with TFT35 E3 V3.0/Configuration.h

```diff
@@ -893,7 +893,7 @@
 
 // Enable this feature if all enabled endstop pins are interrupt-capable.
 // This will remove the need to poll the interrupt pins, saving many CPU cycles.
-//#define ENDSTOP_INTERRUPTS_FEATURE
+#define ENDSTOP_INTERRUPTS_FEATURE
```

Comment:
I’m sure it’s fine, but this is the SKR Mini E3 V1.2 config, not the V3.

Reply:
Yeah - @ellensp and I had a chat after (it was a mistake), but I do believe the v1.2 and v2.0 boards have full support for interrupts on the endstop / probe pins - so I decided to not correct it...

It may well be useful to enable this option across all of the SKR Mini E3 series of boards...

Reply:
> I’m sure it’s fine, but this is the SKR Mini E3 V1.2 config, not the V3.

A question. Is there a reason why interrupts are NOT enabled by default in all configurations? From what I can see from data sheet review, if a pin or processor is incapable of accepting the request, it will be ignored?

On the other hand, enabling interrupt capability can, as the OP suggested, make a very impressive improvement in response time.